src/timer/Timer.js

```diff
@@ -1,17 +1,17 @@
-function Timer(seconds) {
-  this.seconds = seconds;
+class Timer {
+  constructor(seconds) {
+    this.seconds = seconds;
+  }
+  start() {
+    let e = this.seconds;
```

Comment:
Assigning `this.seconds` to `e` isn't necessary when using an arrow function. Do a quick refactor and use `this.seconds` directly in place of `e`